tests/torch-ort_test_api.py

```diff
@@ -0,0 +1,43 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+
+import _test_helpers
+
+import copy
+import onnxruntime as ort
+import torch
+
+from onnxruntime.training import ORTModule
+
+
+class NeuralNetSinglePositionalArgument(torch.nn.Module):
+    def __init__(self, input_size, hidden_size, num_classes):
+        super(NeuralNetSinglePositionalArgument, self).__init__()
+
+        self.fc1 = torch.nn.Linear(input_size, hidden_size)
+        self.relu = torch.nn.ReLU()
+        self.fc2 = torch.nn.Linear(hidden_size, num_classes)
+
+    def forward(self, input1):
+        out = self.fc1(input1)
+        out = self.relu(out)
+        out = self.fc2(out)
+        return out
+
+def test_set_seed():
+    device = 'cuda'
+    N, D_in, H, D_out = 64, 784, 500, 10
+    input = torch.randn(N, D_in, device=device)
```

Comment:
the input and initial weight's randomness are both controlled by torch's seed. 
to test the randomness in ORT, we need a dropout layer.

Reply:
dropout was added. moving the input generation inside the loop makes the test fail

Reply:
yes, input generation is controlled by torch.manual_seed()
If you are interested, you can try reset  torch.manual_seed() in the loop